app/templates/app_menu_list.hbs

```diff
@@ -51,4 +51,4 @@
     </ul>
   </div>
 </div>
-{{partial "sticky_footer"}}
+{{sticky-footer}}
```

Comment:
👍  good move, I think a lot of the partials are more appropriate as components around the app. 
I think we should only use partials to divide templates which are too big, not so much for making reusable pieces